pkg/ui/ccl/src/views/clusterviz/containers/map/nodeCanvas.tsx

```diff
@@ -136,7 +138,18 @@ export class NodeCanvas extends React.Component<NodeCanvasProps, NodeCanvasState
     );
   }
 
+  renderInstructionsBox() {
```

Comment:
given how little content this has, the method doesn't seem to add much value

Reply:
Done.